packages/core-settings/package.js

```diff
@@ -0,0 +1,28 @@
+Package.describe({
+  name: 'unchained:core-settings',
+  version: '0.0.1',
+  // Brief, one-line summary of the package.
+  summary: '',
+  // URL to the Git repository containing the source code for this package.
+  git: '',
+  // By default, Meteor will default to using README.md for documentation.
+  // To avoid submitting documentation, set this field to null.
+  documentation: 'README.md',
```

Comment:
Please see other package.js: A summary and a non-empty readme.md is required to publish it on atmosphere